wiki/Rules/Code_of_conduct_for_modding_and_mapping/en.md

```diff
@@ -4,58 +4,33 @@ The **code of conduct for modding and mapping** is a set of rules and guidelines
-- When pointing out "issues", inspect each thoroughly and consider exactly what it is about the particular "thing" you dislike. Then, try to offer a potential solution or improvement and consider exactly how that suggestion will improve the map. If necessary, try multiple suggestions to see which work best — the more help you can offer the mapper, the better.
-
-Suggestions themselves should ideally consist of four main parts, though based on the experience of the mapper and/or your personal relationship to them, you may want to be more lenient:
-
-1. **A statement of the issue itself and where it can be found** - including a timestamp is a huge help so that the mapper can find the part in question quickly.
-2. **A brief explanation of why this causes a problem** - this is essential in convincing a mapper to even consider your suggestion. Mappers are much more receptive to criticism and change when it's accompanied by clear, understandable and valid reasoning.
-3. **One or more alternative suggestions, accompanied by explanations of how they address the problem(s)** - much like the above, explanation is key here. Even if you can convince the mapper that their current work can be improved, what makes your suggestion better than theirs, or anyone else's?
-4. **The wording (or even language) used should take the mapper into consideration.** Are they a fluent English speaker? If not, stick to simple and easy-to-understand language. If you both speak the same language, by all means write in that language, but try to append an English summary of your mods and mod replies if possible, to ensure everyone stays on the same page. If necessary, support your suggestions with screenshots, illustrations, or even short video clips to help get your point across.
-
-There are any number of ways to present your mod post, so feel free to experiment, but try to keep the visuals simple and formatted in a way that it is easy for the mapper and others to work with.
+- **The goal of modding is to help a mapper improve their work.** If you're not interested in helping people, steer clear of the discussion.
+- **Consider what you're posting, why you're posting it, and who you're posting it to.** This will ensure your best intentions are conveyed and your post is easy to understand.
+  - **Don't shy away from new mappers and modders** — these people are the ones who can benefit the most from your input! Be patient, understanding, and provide as much explanation as possible so that they can learn and improve more quickly
+  - **If you're new yourself, don't be afraid to say so!** This will allow more experienced community members to point you in the right direction. Everyone was new and in your position once!
```

Comment:
I'm not sure if these two subpoints really belong in code of conduct, it feels more like personal/opinionated advice to me at least. helping new people fit in takes work that not everyone wants to commit to and I don't see why that would be wrong from the perspective of a community rules document

Reply:
I was wondering whether leaving them out would be a good idea, decided to maintain them and see what others think.

Personally, either deleting them or adding a general purpose bulletpoint to not antagonize newcoming mappers/modders to discussion could be a nice replacement. I believe having a bulletpoint dedicated to making sure discussions are accessible by any user is important.

Reply:
These rules are *highly* subjective, and the second point is not even a rule - I think there should be something to protect new mappers/modders; maybe "don't assume new mappers are wrong".

Reply:
I agree with Clayton yeah, These probably shouldn't be in modding CoC articles since community member have the freedom to decide if they want to stick with helping their friends or want to extend their hand for other new members, especially with the amount of commitment needed to do this.

However, i do agree that the CoC should still write something that prevent the older/veteran members of the community to underestimate/disregard the opinion from newer member of community, essentially causes friction to happened.

So in my opinion, you should make changes that revolves around preventing people to treating newer people unfairly in modding discussion instead, in order to be more in line with CoC objective.

Reply:
The most recent changes added a new bulletpoint specifically catering towards new player approchability. Please give it a look whenever possible.

Reply:
there is _definitely_ a necessity to be understanding of new people even if you don't want to take the time to walk them through stuff, but yeah I think this new wording definitely approaches it in a way more befitting of a rules/guidelines page. looks good

Reply:
Resolving then, feel free to reopen if more changes are required.